components/alexa/auth_handler.c

```diff
@@ -28,8 +28,9 @@
 
 #define TAG "auth_handler"
 
-
-#define REFRESH_TOKEN CONFIG_ALEXA_AUTH_REFRESH_TOKEN
+#define REFRESH_TOKEN "Atzr|IwEBIEjUTkPyaaA9U3oKJ5FPHs2jVxLU34wcVpRxYxJxyGI85cw5b5TgUy2nknccmfXIMYBmSleKKR14TwXIzwGT7Auahi9RXS_S3NJSCCFCENImuvSMbGeyuEZH9aBybYhDmA1KkMgNFCHovv4eORfck_KKoRWHCeMCH6fD2ZhDy6wxzOahqxbzpa4HY11yQDVJ2q1PM1tjbIT-xTysk785gVdru_itlWiGCb4n-Ip1rfsG9ysMHxG87oTvmjUIoNTNgb_g2u0Ye-ouaKGLdOAGsBhANYfhWYMWgxJDvP9d4qb0NYHwUUSIgyUxF7ZmozXEzZk4kT1xZNY097kYrVvTEznZKc6185V-dDxE0_9_BHIZi0KKBIuPA3gwhAyEBeaeHPfmNo3wPpfbQ6lyScfoFq5eFSXpDukUZk1IUAg6kkli50E6d0ppQSzUONVrsgrgzFL23awD8W4IUunUs6_T-HYzd2_p-SQyD-59ETFwwuo_J0WdCfCimeAKbF8GgAxqlKFW4pjY1AgsBdD3ce61Gu7SBR_NJGrsJtWsQ18KPnAp6A"
```

Comment:
Sorry, i forgot to remove those lines. I am using it to change token without recompiling whole project.